ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -76,43 +72,48 @@ def init_regex_for_range(self, data_directory_path):
         Returns:
             None
         """
-        number_range_df = pd.read_csv(os.path.join(data_directory_path, NUMBER_RANGE_KEYWORD_FILE_NAME),
-                                      encoding='utf-8')
+        number_range_df = pd.read_csv(os.path.join(data_directory_path,
+                                                   numeral_constant.NUMBER_RANGE_KEYWORD_FILE_NAME), encoding='utf-8')
         for index, row in number_range_df.iterrows():
-            range_variants = get_list_from_pipe_sep_string(row[NUMBER_RANGE_FILE_VARIANTS_COLUMN_NAME])
+            range_variants = get_list_from_pipe_sep_string(row[numeral_constant.NUMBER_RANGE_FILE_VARIANTS_COLUMN_NAME])
             for variant in range_variants:
                 self.range_variants_map[variant] = \
-                    NumberRangeVariant(position=row[NUMBER_RANGE_FILE_POSITION_COLUMN_NAME],
-                                       range_type=row[NUMBER_RANGE_FILE_RANGE_TYPE_COLUMN_NAME])
+                    NumberRangeVariant(position=row[numeral_constant.NUMBER_RANGE_FILE_POSITION_COLUMN_NAME],
+                                       range_type=row[numeral_constant.NUMBER_RANGE_FILE_RANGE_TYPE_COLUMN_NAME])
```

Comment:
Minor, we can avoid such long names by using
COLUMN_<ENTITY_NAME>_<COLUMN_NAME>

`NUMBER_RANGE_FILE_RANGE_TYPE_COLUMN_NAME` -> `COLUMN_NUMBER_RANGE_RANGE_TYPE`